src/libcore/iter/range.rs

```diff
@@ -353,6 +304,23 @@ impl<A: Step> Iterator for ops::RangeFrom<A> where
     fn size_hint(&self) -> (usize, Option<usize>) {
         (usize::MAX, None)
     }
+
+    #[inline]
+    fn count(self) -> usize {
+        usize::MAX
```

Comment:
Should this, like `last`, also panic?  Returning a value here is neither of the behaviours (panic or diverge) possible from this today.